### PR TITLE
Fix/arch

### PR DESCRIPTION
**Pull request by n3k00n3**
caesar_cypher by command line and new model.

### CODE REVIEW

**Comment by n3k00n3** [comment]
vigenere done! I need to do text steg again.

**Comment by gjuniioor** [comment]
Questions and suggestions:

* `/src` Isn't better than `/engine`?
* Still need **decrypt** mode
* Instead use `usage()` to show help message, use own `optarg`.
* Have bug when run `$ ruby virgulino.rb --help` or `-h`

**Comment by n3k00n3** [comment]
Gjr, I did what you said. "Engine" i think thats is cool, but i'll to change, all right? I'll to do more changes and set "need revision" here. 
Thanks for u suggestions!

File: engine/virgulino.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+opts.each do |opt, arg| 
+    if opt == '-h' or opt == "--help" 
+        puts help()
+        
+	elsif opt == '-e' or opt == "--encrypt"
+        $encrypt = true
+	
+	elsif opt == '-d' or opt == "--decrypt"
+	     $ncrypt = false
```
Comment (UserXGnu) [inline engine/virgulino.rb]: variable $encrypt is misspelled here ($ncrypt).

**Comment by UserXGnu** [comment]
Due to changes on lampiaosec's team, this pr will be closed.

**Pull request closed without merging**